### PR TITLE
Feature/modules

### PR DESCRIPTION
**Pull request by cynthiamd**

### CODE REVIEW

File: packages/gatsby-theme-basic/src/components/Card.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ Card.propTypes = {
   as: PropTypes.elementType,
   children: PropTypes.node,
   className: PropTypes.string,
+  contentType: PropTypes.string,
```
Comment (jeanfredrik) [inline packages/gatsby-theme-basic/src/components/Card.js:11]: Please go through all changes and remove unused code like this.

**Review by jeanfredrik — COMMENTED** [review]
Can you provide instructions on how to test this PR?

**Comment by cynthiamd** [comment]
How to test this pr

Page link with all modules:
http: //localhost: 8000/cache-for-web-editors /modules/

Color blocks
Add a post module and select "Index" as the display and "colored blocks" as the layout.
Try creating manual posts or selecting existing posts.
When selecting existing posts, it must be possible to show the date and excerpt. The theme colors should be global for all cards.
When choosing manual posts, you should be able to set theme colors for each card. You should also be able to add permalinks.

Post module
Create different variants of the post module with title, without title.
When selecting "grid" or "mixedgrid" as the layout, the archive link must also be visible in the heading for the item types news and events.
Try clicking on the tags.

Text module
Create a text module with different theme colors, without box-frame, with box-frame, without title and with title.
NOTE. the styling of the headlines is not finalized.

Image module
Add an image module with links and an image caption (field: image caption).
Try leaving the image caption field empty and fill in the image's "Caption" to see if it appears.
Try filling in the "photographer" field and the alt field.

Note module
Create a note module with different statuses, with title and without title. When there is no title, the icon should appear next to the text and vice versa when there is a title.

The table module
Create a table module with different theme colors, with heading and without heading. Test in mobile mode.
NOTE. there is no support for uploading csv files.

The file module
Create a movie module with and without a title. Test in mobile mode.

Content pages
Post:
Publication date, title, update date below the content
Page:
Children's pages and sibling pages are visible and match the styling. Update date can be seen below the content.
Event:
Update date can be seen below the content.

**Comment by jeanfredrik** [comment]
Perfect! Thank you. I'll test and get back to you

**Review by jeanfredrik — APPROVED** [review]
I've made several changes. Biggest of them being refactoring of the Posts module. I'm merging this now since it’s grown very large. Additional work should be made in new PRs.

**Pull request merged**